src/server/controllers/UserController.ts

```diff
@@ -60,7 +60,14 @@ export class UserController implements UserControllerInterface {
     req: Express.Request,
     res: Express.Response,
   ) => Promise<void> = async (req, res) => {
-    const { userId, longUrl, shortUrl, state }: UrlEditRequest = req.body
+    const {
+      userId,
+      longUrl,
+      shortUrl,
+      state,
+      description,
+      contactEmail,
+    }: UrlEditRequest = req.body
     const file = req.files?.file
```

Comment:
can we refactor this function to introduce utility functions in the url management service for each of these items that may potentially be updated? i see a lot of inline logic in the controller.

Reply:
Do you mean moving the preprocessing to the service layer?